src/main/java/net/pms/encoders/FFMpegVideo.java

```diff
@@ -266,14 +266,20 @@ public FFMpegVideo(PmsConfiguration configuration) {
 			filterChain.add(overrideVF);
 		}
 
-		// Convert 3D video to the other output 3D format
+		// Convert 3D video to the other output 3D format or to 2D using "Output3DFormat = ml" or "Output3DFormat = mr" in the renderer conf
+		String stereoLayout = null;
+		if (media.get3DLayout() != null) {
+			stereoLayout = media.get3DLayout().toString().toLowerCase();
```

Comment:
@valib You should use ```.toLowerCase(Locale.ROOT)``` on all ```toLower``` or ```toUpper``` conversions that you want to use "English" rules instead of localized rules. The best example of why is the Turkish locale where ```i``` and ```I``` are different letters and won't be changed as you expect if using localized rules.